CHANGELOG.md

```diff
@@ -1,69 +1,102 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.2.0] - Septempber, 8, 2022
```

Comment:
We have previously set the documentation site changes separately here. If we go with that, we should probably include the documentation changes to the pagination component and table component? 🤔